processing/clinical.py

```diff
@@ -370,8 +370,9 @@ def validate_helper(self, clinicalDF, clinicalSampleDF, oncotreeLink):
 				total_error += "Sample: There can't be any blank values for PATIENT_ID\n"
 
 			#CHECK: within the sample file that the sample ids match the patient ids
-			if not all([patient in sample for sample, patient in zip(clinicalSampleDF[sampleId], clinicalSampleDF[patientId])]):
-				total_error += "Sample: PATIENT_ID's much be contained in the SAMPLE_ID's (ex. SAGE-1 <-> SAGE-1-2)\n"
+			if haveSampleColumn:
```

Comment:
do you need this?  doesn't the else already say that there IS a sample column?

Reply:
oops.  my mistake.  the github review made it look like this is right under line 247.